### ISSUE TITLE
LoadError: cannot load such file -- pry

### ISSUE THREAD

**Issue by ruby-fu-ninja** [ilyakatz/data-migrate#95]
Stack trace:

```
LoadError: cannot load such file -- pry
/home/deploy/apps/app/shared/bundle/ruby/2.3.0/gems/activesupport-4.2.10/lib/active_support/dependencies.rb:274:in `require'
/home/deploy/apps/app/shared/bundle/ruby/2.3.0/gems/activesupport-4.2.10/lib/active_support/dependencies.rb:274:in `block in require'
/home/deploy/apps/app/shared/bundle/ruby/2.3.0/gems/activesupport-4.2.10/lib/active_support/dependencies.rb:240:in `load_dependency'
/home/deploy/apps/app/shared/bundle/ruby/2.3.0/gems/activesupport-4.2.10/lib/active_support/dependencies.rb:274:in `require'
/home/deploy/apps/app/shared/bundle/ruby/2.3.0/gems/data_migrate-5.0.0/tasks/databases.rake:2:in `<top (required)>'
```

Affecting version `5.0.0` but rolling back to `~> 4.0.0` resolves the issue

**Comment by EnomaDebby** [comment]
Hi @ruby-fu-ninja did you try running a bundle install and still encountered this issue on version 5.0.0?

**Comment by CharlesMcMillan** [comment]
We are seeing this as well. We do not include 'pry' in our production environments (RAILS_ENV=production), so the require 'pry' blows up in any prod environment

**Comment by ilyakatz** [comment]
Sorry about this, I forgot to remove this. @EnomaDebby added a fix in #96 , will be deployed shortly

**Comment by EnomaDebby** [comment]
Fix [#96](https://github.com/ilyakatz/data-migrate/pull/96) deployed

**Comment by ilyakatz** [comment]
This is fixed by @EnomaDebby
